tests/sentry/models/test_groupsnooze2.py

```diff
@@ -0,0 +1,303 @@
+import functools
+import itertools
+import random
+from datetime import datetime, timedelta
+from typing import Optional, Sequence
+from unittest import mock
+
+import pytest
+import pytz
+from django.utils import timezone
+
+from sentry import tsdb
+from sentry.event_manager import _pull_out_data
+from sentry.models import Group, GroupSnooze
+from sentry.testutils import SnubaTestCase, TestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+from sentry.testutils.silo import region_silo_test
+from sentry.types.issues import GroupType
+from sentry.utils.samples import load_data
+
+### This file is purely for debugging. I'll delete it before the PR is merged. ###
```

Comment:
This is just a big ol file of me losing my mind trying to figure out what's going on with these tests. I took it out for now cause I wasn't getting anywhere, but I was swapping out the tsdb's in a previous commit.